ci/docker-rust-nightly/README.md

```diff
@@ -16,6 +16,40 @@ To update the pinned version:
 1. Run `ci/docker-rust-nightly/build.sh` to rebuild the nightly image locally
 1. Run `ci/docker-run.sh --nopull solanalabs/rust-nightly ci/test-nightly.sh` to
    confirm the new nightly image builds.  Fix any issues as needed
+1. Run `docker login` to enable pushing images to Docker Hub, if you're authorized.
 1. Run `CI=true ci/docker-rust-nightly/build.sh` to push the new nightly image to dockerhub.com.
    **Note that all CI may be broken now until the next step is completed.**
 1. Merge the update to `ARG nightly=xyz` and any codebase adjustments needed.
+
+
+## Troubleshooting
+
+### Permission denied
```

Comment:
nit: I'd just remove this section.   It's now documented at https://github.com/solana-labs/solana/commit/b4a7a183345e3cd7e654e0a7d596f2684bcdb0cd (rebase this PR on the tip of master)